### PR TITLE
[WIP] Do per repeat per leg results gathering

### PR DESCRIPTION
**Pull request by IAlibay**
TODO:
- [ ] Fix tests

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: openfecli/commands/gather.py [hunk 1]
```diff
@@ -76,6 +77,21 @@ def get_type(res):
         return 'solvent'
 
 
+def get_unit_results(res):
+    unit_results = defaultdict(dict)
+
+    for key in res['unit_results']:
+        dg = res['unit_results'][key]['outputs']['unit_estimate']
+        mbar_error = res['unit_results'][key]['outputs']['unit_estimate_error']
```
Comment (IAlibay) [inline openfecli/commands/gather.py:85]: This makes things not backwards compatible, do we care?

**Comment by IAlibay** [comment]
@mikemhenry we're going to need a new version of `rbfe_results.tar.gz` - is this something you might be able to generate?

**Comment by mikemhenry** [comment]
@IAlibay Yes, do you want me to make it with this branch?

**Comment by IAlibay** [comment]
> @IAlibay Yes, do you want me to make it with this branch? 

I don't _think_ anything I've gone here will affect simulations, just the gathering. If we just do another run using the current main branch that should be fine.

**Comment by mikemhenry** [comment]
Cool, doing that right now, assuming this commit is recent enough e2eeb85356a5426a3d54aa2e61f6b01212ac770e

**Comment by IAlibay** [comment]
> Cool, doing that right now, assuming this commit is recent enough e2eeb85356a5426a3d54aa2e61f6b01212ac770e

@mikemhenry did you manage to generate these results?

**Comment by richardjgowers** [comment]
done in #620

**Pull request closed without merging**